new-client/src/plugins/Sketch/components/featureStyle/FeatureStyleSelector.js

```diff
@@ -1,12 +1,36 @@
 import React from "react";
 import { Grid, Typography, TextField } from "@mui/material";
-import { STROKE_DASHES } from "plugins/Sketch/constants";
+import {
+  STROKE_DASHES,
+  DEFAULT_DRAW_STYLE_SETTINGS,
+} from "plugins/Sketch/constants";
 
 import FeatureStyleAccordion from "./FeatureStyleAccordion";
 import FeaturePointSizeAccordion from "./FeatureSizeAccordion";
 import StrokeTypeSelector from "./StrokeTypeSelector";
 
 export default function FeatureStyleSelector(props) {
+  // Since the strokeType value "none" is only available for the activeDrawType "Circle" and "Polygon" we Automatically want to set the strokeType to the default value "solid"
+  // if activeDrawType is neither "Circle" or "Polygon" and strokeType is "none"
+  React.useEffect(() => {
+    if (
+      !["Circle", "Polygon"]?.includes(props.activeDrawType) &&
+      props.drawStyle.strokeType === "none"
+    ) {
+      // Update drawStyle with solid strokeType
+      props.setDrawStyle((prevDrawStyle) => ({
+        ...prevDrawStyle,
+        strokeType: DEFAULT_DRAW_STYLE_SETTINGS.strokeType,
+        lineDash: STROKE_DASHES.get(DEFAULT_DRAW_STYLE_SETTINGS.strokeType),
+        strokeColor: {
+          ...props.drawStyle.strokeColor,
+          a: DEFAULT_DRAW_STYLE_SETTINGS.strokeColor.a,
+        },
+      }));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.activeDrawType, props.drawStyle]);
```

Comment:
I would suggest that you destructure the props outside the useEffect and remove the eslint-disable row!

Something like:

```js
const {activeDrawtype, drawStyle} = props;

useEffect(() => {...}, [activeDrawtype, drawStyle])

Reply:
Cheers! I'll make sure to remove the eslint-disable rows in #1310 also.